### PR TITLE
Disable policy deployment by default

### PR DESCRIPTION
**Pull request by howardjohn**

### CODE REVIEW

**Comment by howardjohn** [comment]
Needed to add a new default "integ test" (as opposed to e2e/) default values since it hardcoded policy to enabled

File: operator/data/profiles/default.yaml [hunk 1]
```diff
@@ -54,7 +54,7 @@ spec:
 
   # Policy feature
     policy:
-      enabled: true
+      enabled: false
```
Comment (mandarjog) [inline operator/data/profiles/default.yaml]: Assuming this controls the deployment itself, lgtm

@douglas-reid ?

**Review by elfinhe — APPROVED** [review]

File: operator/data/profiles/default.yaml (hunk 1 above)
Comment (howardjohn) [reply operator/data/profiles/default.yaml]: Yes, this is controls the deployment

**Review by mandarjog — APPROVED** [review]
Lgtm

**Pull request merged**